src/providers/tfc/workspaceProvider.ts

```diff
@@ -117,6 +117,8 @@ export class WorkspaceTreeDataProvider implements vscode.TreeDataProvider<Worksp
           include: ['current_run'],
           // Include query parameter only if project filter is set
           ...(this.projectFilter && { 'filter[project][id]': this.projectFilter }),
+          'page[size]': 50,
+          sort: '-current-run.created-at',
```

Comment:
Not completely sold on this changing the order on refresh, but I do like the experience.